scalability_test/pvc_template.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ope-placeholder
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 4Gi
```

Comment:
Can this be smaller? Should it be configurable?

Reply:
Yes, it could be as small as 1Gi.